debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: argobots
+Source: <url://example.com>
```

Comment:
Should this be filled out?

Reply:
I have filled it out.  I was leaving it as a template for our internal use.